clojure-mode.el

```diff
@@ -192,6 +192,11 @@ Out-of-the box `clojure-mode' understands lein, boot and gradle."
           (and (listp value)
                (cl-every 'stringp value))))
 
+(defcustom clojure-project-root-locating-function nil
```

Comment:
The defcustom should also have a `:package-version` property pointing this was added in version 5.7.0.